support/k8s/k8s-workload-registrar/webhookhandler_test.go

```diff
@@ -3,6 +3,7 @@ package main
 import (
 	"context"
 	"errors"
+	admv1 "k8s.io/api/admission/v1"
```

Comment:
This needs to go in the second list where admv1beta1 used to be.

Reply:
thank you @faisal-memon, fixed.